### ISSUE TITLE
solo ranked match, whatever my side is the team where i am always gets the first pick

### ISSUE THREAD

**Issue by hiromi0328** [ValveSoftware/Dota2-Gameplay#2199]
### Description

solo ranked match, whatever my side is the team where i am always gets the first pick EVERY SINGLE TIME,
i used to dont mind it for almost a year now, but now im actually starting to get serious ranking up, but every game starts whatever side im on, where i am always gets to pick first, and it always puts me in a disadavantage as my side always gets counter picked. 

anyone has this issue?

screen shot provided, ign- SovietOnion
![Screenshot 2022-08-22 170626](https://user-images.githubusercontent.com/111738934/185883950-13109e7b-45b3-4213-aa50-0afb1daffd62.png)

### Screenshots

![Screenshot 2022-08-22 170626](https://user-images.githubusercontent.com/111738934/185884047-4979914a-c21c-4ee0-ba27-340365854cc3.png)

**Comment by Farzam2003** [comment]
2 games is not always and beside they dont see your picks, your first 2 pick and then they pick 2 heroes and then heroes are revealed

**Comment by hiromi0328** [comment]
> 2 games is not always and beside they dont see your picks, your first 2 pick and then they pick 2 heroes and then heroes are revealed

No, we get to pick first 2, and then reveal, then countdown starts for picking on the other team. And its not just 2 games. It has been like this for a year already.

**Comment by hiromi0328** [comment]
It is ranked roles btw

**Comment by Farzam2003** [comment]
> > 2 games is not always and beside they dont see your picks, your first 2 pick and then they pick 2 heroes and then heroes are revealed
> 
> 
> 
> No, we get to pick first 2, and then reveal, then countdown starts for picking on the other team. And its not just 2 games. It has been like this for a year already.

they dont see your teams pick ban the system was around for 2 years how do you still not know this? the whole first pick last pick thing is gone since 7.24 or sth

**Comment by AzureSacri** [comment]
2 games is not always and beside they dont see your picks, your first 2 pick and then they pick 2 heroes and then heroes are revealed [mcdvoice](https://mcdvoice.me/)

**Comment by MontezD11** [comment]
Thanks for the information, I will try to figure it out for more. Keep sharing such informative post keep suggesting such post.

**Comment by CatLovesBest** [comment]
Thanks for this. [CatLovesBest](https://www.catlovesbest.com/)

**Comment by CatLovesBest** [comment]
I appreciate the information and advice you have shared.
[https://www.catlovesbest.com/](url)

**Comment by Stlef1972** [comment]
Thank you guys.

**Comment by Eric78o** [comment]
If you disagree with someone else's comment, engage in a constructive conversation rather than attacking the person personally.
